pymc3/distributions/logprob.py

```diff
@@ -259,7 +265,14 @@ def _logp(
 
     """
     value_var = rvs_to_values.get(var, var)
-    return at.zeros_like(value_var)
+    _values = {k: at.as_tensor_variable(e) for k, e in rvs_to_values.items()}
+    logp_fn = joint_logprob(var, _values, **kwargs)
```

Comment:
This way the `Op`s with no registered logp's in PyMC can 'fall back' to their aeppl implementation and can be replaced with appropriate RVs (For instance the current Mixture will be a `Subtensor` later replaced by `MixtureRV` and also the upcoming `Scan` which will be replaced by `ScanRV` )

Is this a good entry point for `joint_logprob`? cc @brandonwillard

Reply:
Something like that could work for now, but keep on mind that we'll probably want to entirely remove this module at some point after the integration is established.